Classes/Hooks/ImageTextLinkPreviewRenderer.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace T3kit\themeT3kit\Hooks;
+
+class ImageTextLinkPreviewRenderer implements \TYPO3\CMS\Backend\View\PageLayoutViewDrawItemHookInterface
+{
+	/**
+	 * Preprocesses the preview rendering of the custom content element which has image and media image
+	 *
+	 * @param \TYPO3\CMS\Backend\View\PageLayoutView $parentObject Calling parent object
+	 * @param bool $drawItem Whether to draw the item using the default functionalities
+	 * @param string $headerContent Header content
+	 * @param string $itemContent Item content
+	 * @param array $row Record row of tt_content
+	 * @return void
+	 */
+	public function preProcess(\TYPO3\CMS\Backend\View\PageLayoutView &$parentObject, &$drawItem, &$headerContent, &$itemContent, array &$row) {
```

Comment:
@chantheoun is there a reason for the full NS path? Could you add `use` statement before introducing the class?